src/main/java/com/entropyteam/entropay/employees/controllers/HolidayController.java

```diff
@@ -46,17 +40,4 @@ public ResponseEntity<List<Map<String, Integer>>> getHolidayYears() {
         return ResponseEntity.ok().header(BaseController.X_TOTAL_COUNT, String.valueOf(holidayYears.size()))
                 .body(holidayYears);
     }
-
-    @GetMapping("/sync-holiday")
```

Comment:
This method was to create in the calendar the holidays that already exist in the database.